docs/troubleshooting/deployment/codemagic_deployment_tips.md

```diff
@@ -0,0 +1,16 @@
+---
+keywords: ['deployment', 'tips', 'codemagic']
+slug: /codemagic-deployment-tips
+title: CodeMagic Deployment Tips
+---
+# CodeMagic Deployment Tips
+
+Here are some tips to avoid Deployment issues:
+
+:::tip
+- Make sure you've followed all the steps for [setting up deployment](http://localhost:3000/deployment/deploy-for-environments/#mobile-deployment)
```

Comment:
I gave the link as a reference, why are we using a localhost link? it wont work in production obviously. please reference other docs to see how to reference links properly. also follow the contribution guidelines, hyperlinks are bold always, i've mentioned this before.